lib/bin.js

```diff
@@ -2,39 +2,62 @@
 
 'use strict';
 var yargs = require('yargs')
-   .boolean('h')
-  .alias('h', 'help')
-  .describe('h', 'this help message')
-  .default('p', 5080)
-  .alias('p', 'port')
-  .describe('p', 'port')
-  .default('P', 16984)
-  .alias('P', 'pouch-port')
-  .describe('P', 'pouchdb-server port')
-  .alias('l', 'log')
-  .describe('l', 'error|warn|info|debug')
-  .default('l', 'info')
-  .default('r', 'https://registry.npmjs.org')
-  .alias('r', 'remote')
-  .describe('r', 'remote fullfatdb')
-  .default('R', 'https://skimdb.npmjs.com/registry')
-  .alias('R', 'remote-skim')
-  .describe('R', 'remote skimdb')
-  .default('u', 'http://127.0.0.1:5080')
-  .alias('u', 'url-base')
-  .describe('u', 'base url it will be hosted on')
-  .default('d', './')
-  .alias('d', 'directory')
-  .describe('directory', 'directory to store data')
+  .option('h', {
+      alias: 'help',
+      type: 'boolean',
+      describe: 'this help message'
+  })
+  .option('p', {
+      'default': 5080,
```

Comment:
Do all these `'default'` values need to be stringified because `default` is a javascript keyword? Will something bad happen if they're not stringified?